kotlin-analysis/intellij-dependency/build.gradle.kts

```diff
@@ -30,19 +21,19 @@ fun jpsModel() = zipTree(jpsStandalone.singleFile).matching {
 }
 
 dependencies {
-    val kotlin_plugin_version: String by project
+    val kotlin_plugin_version: String = libs.versions.kotlin.plugin.get()
+
     api("org.jetbrains.kotlin:common:$kotlin_plugin_version")
     api("org.jetbrains.kotlin:idea:$kotlin_plugin_version") {
         isTransitive = false
     }
     api("org.jetbrains.kotlin:core:$kotlin_plugin_version")
     api("org.jetbrains.kotlin:native:$kotlin_plugin_version")
```

Comment:
Why not put these into toml?

Reply:
this PR is still WIP (but I forgot to mark it). I got started with the Version Catalog in #2704, but decided to split it out and I made this PR so I could put a link next to the TODO https://github.com/Kotlin/dokka/pull/2704/files#diff-7b5c40eb5522b5216cfe8c75612e579a8db0daee9ded80df9ce6a56e8361a962R11